blocklylib-core/src/main/java/com/google/blockly/android/ui/VirtualWorkspaceView.java

```diff
@@ -178,6 +178,24 @@ public void setScrollable(boolean scrollable) {
     }
 
     /**
+     * Configures whether the user can scale the workspace by touch events.
+     *
+     * @param scalable Allow scalability if true. Otherwise, disable it.
+     */
+    public void setScalable(boolean scalable){
```

Comment:
Replace this and setScrollable with setZoomBehavior().
Instead of removing the mScaleGestureDetector, use mZoomBehavior to determine what to update within ScaleGestureDetector#onScale().